src/palace/manager/feed/annotator/circulation.py

```diff
@@ -740,6 +740,18 @@ def __init__(
         self._top_level_title = top_level_title
         self.identifies_patrons = library_identifies_patrons
         self.facets = facets or None
+        self._is_novelist_configured: bool | None = None
+
+    @property
+    def is_novelist_configured(self) -> bool:
+        """Lazy load and cache NoveList's `is_configured` flag.
+
+        This is an optimization to avoid a SQL query to check this
+        flag for every book in the feed.
+        """
+        if self._is_novelist_configured is None:
+            self._is_novelist_configured = NoveListAPI.is_configured(self.library)
```

Comment:
Minor: Should `is_configured` maybe be private to make it clear the property should be the preferred way to access?

Reply:
I didn't do that because there was a class method that used `is_configured`, but could not access the instance property. I'll take another look to see if I can clean that up, too.

Reply:
I decided not to make it private, since it's part of the public api of NoveListAPI. I added a docstring to indicate that there can be performance implications to running it. I also renamed it to hint in that direction, as well.